components/Button/src/button.css

```diff
@@ -1,129 +1,144 @@
+  padding-inline-end: var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline));
+  padding-inline-start: var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline));
+  position: relative;
 }
 
 .denhaag-button--focus:after,
 .denhaag-button:focus:after {
+  border: var(--denhaag-focus-border);
+  border-radius: var(--denhaag-border-radius);
+  bottom: -2px;
   content: "";
   display: block;
-  position: absolute;
-  top: -2px;
-  bottom: -2px;
   left: -2px;
+  position: absolute;
   right: -2px;
-  border: var(--denhaag-focus-border);
-  border-radius: var(--denhaag-border-radius);
+  top: -2px;
 }
 
 /* Due to the accessibility focus outline overlapping with the border, the outline needs to be repositioned for the border variant */
 .denhaag-button--secondary-action.denhaag-button--focus:after,
 .denhaag-button--secondary-action:focus:after {
-  top: -3px;
   bottom: -3px;
   left: -3px;
   right: -3px;
+  top: -3px;
 }
 
-.denhaag-button--primary-action {
-  background-color: var(--denhaag-button-primary-action-background-color);
-  color: var(--denhaag-button-primary-action-color);
-}
-
-.denhaag-button--primary-action.denhaag-button--hover,
-.denhaag-button--primary-action:hover {
+.denhaag-button.denhaag-button--hover,
+.denhaag-button:hover {
   background-color: var(--denhaag-button-primary-action-hover-background-color);
   color: var(--denhaag-button-primary-action-hover-color);
 }
 
-.denhaag-button--primary-action.denhaag-button--disabled,
-.denhaag-button--primary-action.denhaag-button:disabled {
+.denhaag-button.denhaag-button--disabled,
+.denhaag-button.denhaag-button:disabled {
   background-color: var(--denhaag-button-primary-action-disabled-background-color);
   color: var(--denhaag-button-primary-action-disabled-color);
 }
 
 .denhaag-button--secondary-action {
   background-color: var(--denhaag-button-secondary-action-background-color);
-  color: var(--denhaag-button-secondary-action-color);
-  border-width: var(--denhaag-button-border-width);
-  border-style: solid;
   border-color: var(--denhaag-button-secondary-action-border-color);
+  border-style: solid;
+  border-width: var(--denhaag-button-border-width);
+  color: var(--denhaag-button-secondary-action-color);
+  padding-block-end: calc(
+    var(--denhaag-button-medium-size-padding-block, var(--denhaag-button-padding-block)) -
+    var(--denhaag-button-border-width)
+  );
+  padding-block-start: calc(
+    var(--denhaag-button-medium-size-padding-block, var(--denhaag-button-padding-block)) -
+    var(--denhaag-button-border-width)
+  );
+  padding-inline-end: calc(
+    var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline)) -
+    var(--denhaag-button-border-width)
+  );
+  padding-inline-start: calc(
+    var(--denhaag-button-medium-size-padding-inline, var(--denhaag-button-padding-inline)) -
+    var(--denhaag-button-border-width)
+  );
```

Comment:
Could you explain this one? Seems a bit counter-intuitive to me that when I specify a padding of a certain value the padding set will be that minus the border-width. Not sure this is really user-friendly.

Reply:
Ah I see, we already had that one to ensure no size differences between the secondary action and the primary action button (which has no border), nvm me then